polars/polars-core/src/chunked_array/ops/unique/mod.rs

```diff
@@ -61,6 +63,11 @@ impl<T: PolarsObject> ChunkUnique<ObjectType<T>> for ObjectChunked<T> {
     fn arg_unique(&self) -> PolarsResult<IdxCa> {
         polars_bail!(opq = arg_unique, self.dtype());
     }
+
+    fn estimate_approx(&self, mut e: HyperLogLogPlus<AnyValue, RandomState>) -> f64 {
```

Comment:
The object type should not implement this.

Reply:
done.